test/test_false_rumors.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import time
+
+servers_list = [13301, 13302, 13303]
+
+
+def check_rumors(srv, expected):
+    assert srv.conn.eval("return rumors")[0] == expected
+
+
+def test_setup(servers, helpers):
+    assert servers[13301].conn.eval('''
+        rumors = setmetatable({ }, {__serialize = 'map'})
+
+        local fiber = require('fiber')
+        local members = require('membership.members')
+        local opts = require('membership.options')
+
+        local function collect_rumors()
+            for uri, m in members.pairs() do
+                if m.status ~= opts.ALIVE then
+                    rumors[uri] = opts.STATUS_NAMES[m.status]
+                end
+            end
+        end
+
+        _G._protection_fiber = fiber.create(function()
+            local cond = membership.subscribe()
+            while true do
+                cond:wait()
+                fiber.testcancel()
+                collect_rumors()
+            end
+        end)
```

Comment:
It's worth to describe what this monkeypatch does.

Reply:
Done